internal/bundle/lint.go

```diff
@@ -34,6 +34,7 @@ const (
 	queryTitle                 = "query-name"
 	queryUidUnique             = "query-uid-unique"
 	queryUnassigned            = "query-unassigned"
+	queryUsedAsDifferentTypes  = "query-used-as-different-types"
```

Comment:
i think you need to update the `LinterRules` var with this entry too so that it shows in the sarif report

Reply:
good point! I missed that